### PR TITLE
Add user moiety & created_at to API

### PR DESCRIPTION
**Pull request by Throne3d**
Moieties requested by DanielH, created_at to give parity with the user index of the UI. Builds on #1286 to reduce diffs.

### CODE REVIEW

File: app/presenters/user_presenter.rb [hunk 1]
```diff
@@ -8,6 +8,7 @@ def initialize(user)
   def as_json(options={})
     return {} unless user
     return {id: user.id, username: '(deleted user)'} if user.deleted?
-    user.as_json_without_presenter({only: [:id, :username]}.reverse_merge(options))
+    attrs = %w(id username moiety moiety_name created_at)
```
Comment (Marri) [inline app/presenters/user_presenter.rb]: given my general feelings about moieties, could this maybe be returned in... many fewer places xD

Comment (Throne3d) [reply app/presenters/user_presenter.rb]: I couldn't figure out where we return users – I checked for explicit `to_json` calls. Would I have to check the whole API? :/

Comment (Throne3d) [reply app/presenters/user_presenter.rb]: Oh, I guess maybe our examples are comprehensive enough that I can focus on those?

Comment (Throne3d) [reply app/presenters/user_presenter.rb]: Is this better? I put it in users#index only, but it still shows up three times.

**Pull request merged**